Pod/Classes/MenuItemView.swift

```diff
@@ -145,6 +178,18 @@ public class MenuItemView: UIView {
         translatesAutoresizingMaskIntoConstraints = false
     }
     
+    private func setupLabels(title: String, desc: String) {
+        titleLabel.text = title
+        titleLabel.textColor = options.textColor
+        titleLabel.font = options.font
+        addSubview(titleLabel)
```

Comment:
Make `setupTitleLabel` and `setupDescriptionLabel`, then you can call `setupTitleLabel` in `setupDescriptionLabel`.